### ISSUE TITLE
Update Auth README to make more clear description

### ISSUE THREAD

**Issue by dakeshi** [firebase/FirebaseUI-iOS#102]
`kGoogleClientID`, `kFacebookAppID` look like a global constant in [auth docs example code.](https://github.com/firebase/FirebaseUI-iOS/tree/master/FirebaseUI/Auth)
It would be ambiguous because it could be confused with the usage of Google sign-in library.

Refer to [enable sign section in Google sign-in docs](https://developers.google.com/identity/sign-in/ios/sign-in)

>Note: If you manually installed the SDK, you should instead configure the GIDSignIn object directly, using the client ID found in the GoogleService-Info.plist file: 
[GIDSignIn sharedInstance].clientID = kClientID;

Most of users would install firebaseUI through CocoaPods, so we don't use `kGoogleClientID` or `kFacebookAppID` as a special constant, such as `kClientID` in Google sign-in.

So, I think that it should be updated an example code.
#95 is a good example.

```swift
let kGoogleAppClientID = (FIRApp.defaultApp()?.options.clientID)!

// Your Facebook App ID, which can be found on developers.facebook.com.
let kFacebookAppID     = "FaceBook App ID"
```

**Comment by yramanchuk** [comment]
I propose to get rid of `facebookAppId` and `googleClientId` as initialization params of FirebaseUI providers.

- GoogleClientId can be retrieved by FirebaseUI Google provider.
- Facebook App Id can be specified in info.plist (https://developers.facebook.com/docs/ios/getting-started#configure-xcode-project) - later FirebaseUI Facebook provider will retrieve from it

**Comment by yramanchuk** [comment]
`Google Client ID` and `Facebook AppId` were removed from initialization params of FirebaseUI providers (providers are reading them internally).

@dakeshi this fix will be included in v0.5.6
If needed I can describe how to test it locally.

**Comment by dakeshi** [comment]
Thanks. @yramanchuk It looks more nice approach.
